examples/panel-basic/README.md

```diff
@@ -16,30 +20,32 @@ Use panel plugins when you want to do things like visualize data returned by dat
    yarn install
    ```
 
-2. Build plugin in development mode and run inside Grafana using Docker:
+2. Build the plugin in development mode and run inside Grafana using Docker:
+
 
    ```bash
    # Start watching for changes
-   yarn dev
+   npm dev
```

Comment:
Except for `npm install` all of these should `npm run [command]` or we'll get an error.

here should be `npm run dev`  and down the line `npm run build` and `npm run e2e`